osmoutils/coin_helper.go

```diff
@@ -127,3 +127,16 @@ func ConvertCoinArrayToCoins(coinArray []sdk.Coin) sdk.Coins {
 	}
 	return coins
 }
+
+func ConvertCoinsToCoinArray(coins sdk.Coins) []sdk.Coin {
```

Comment:
Im pretty sure you can do a type cast for this, seeing as `sdk.Coins` is just a type alias.

Reply:
You are correct, calling []sdk.Coin(coinsObject) works, will implement this instead

Reply:
Done here https://github.com/osmosis-labs/osmosis/pull/7240/commits/ba0c68eddd73c01e650e2852d3f65029a81e38a2